docs/api.md

```diff
@@ -0,0 +1,3827 @@
+# ArduinoBLE library
+
+## BLE class
+
+Used to enable the BLE module.
+
+### `begin()`
+
+Initializes the BLE device.
+
+#### Syntax
+
+```
+BLE.begin()
+
+```
+
+#### Parameters
+
+None
+
+Returns
+1 on success, 0 on failure.
+
+#### Example
+
+```Arduino
+…
```

Comment:
Two things to fix:
- it needs to be "arduino" instead of "Arduino" before the code snippet (after the backticks)
- remove the "…" from code snippets. They occur at the top & bottom of each sketch.

Reply:
done